### PR TITLE
Prevent scrolling of fullScreen Popup to focused input in Safari (T714801)

### PR DESCRIPTION
**Pull request by alexander-kotov-dx**

### CODE REVIEW

File: js/ui/overlay/ui.overlay.js [hunk 1]
```diff
@@ -1248,7 +1253,25 @@ var Overlay = Widget.inherit({
         var $wrapper = this._$wrapper,
             $container = this._getContainer();
 
-        $wrapper.css("position", this._isWindow($container) && !iOS ? "fixed" : "absolute");
+        $wrapper.css("position", this._isWindow($container) && this._fixedPositionEnable() ? "fixed" : "absolute");
+    },
+
+    _fixedPositionEnable: function() {
+        return !iOS || this.bodyScrollTop !== undefined;
```
Comment (San4es) [inline js/ui/overlay/ui.overlay.js]: `bodyScrollTop` -> `_bodyScrollTop`
`_fixedPositionEnable` -> `_shouldFixedPositionBeUsed`/`_canFixedPositionBeUsed`/`_useFixedPosition`/etc.?

File: js/ui/overlay/ui.overlay.js [hunk 2]
```diff
@@ -1248,7 +1253,25 @@ var Overlay = Widget.inherit({
         var $wrapper = this._$wrapper,
             $container = this._getContainer();
 
-        $wrapper.css("position", this._isWindow($container) && !iOS ? "fixed" : "absolute");
+        $wrapper.css("position", this._isWindow($container) && this._useFixedPosition() ? "fixed" : "absolute");
```
Comment (DokaRus) [inline js/ui/overlay/ui.overlay.js]: Extract value to the separate variable

File: js/ui/overlay/ui.overlay.js [hunk 3]
```diff
@@ -1245,10 +1250,28 @@ var Overlay = Widget.inherit({
     },
 
     _fixWrapperPosition: function() {
-        var $wrapper = this._$wrapper,
-            $container = this._getContainer();
+        var $wrapper = this._$wrapper;
```
Comment (DokaRus) [inline js/ui/overlay/ui.overlay.js]:
```suggestion

```

File: js/ui/overlay/ui.overlay.js [hunk 4]
```diff
@@ -1245,10 +1250,28 @@ var Overlay = Widget.inherit({
...
+
```
Comment (DokaRus) [inline js/ui/overlay/ui.overlay.js]:
```suggestion

```

File: js/ui/overlay/ui.overlay.js [hunk 5]
```diff
@@ -1245,10 +1250,28 @@ var Overlay = Widget.inherit({
...
+        $wrapper.css("position", this._useFixedPosition() ? "fixed" : "absolute");
```
Comment (DokaRus) [inline js/ui/overlay/ui.overlay.js]:
```suggestion
        this._$wrapper.css("position", this._useFixedPosition() ? "fixed" : "absolute");
```

**Review by DokaRus — APPROVED** [review]

**Review by San4es — APPROVED** [review]

**Pull request merged**